SimpleApp/ViewController.swift

```diff
@@ -2,34 +2,124 @@ import UIKit
 
 class ViewController: UIViewController {
     @IBOutlet private(set) var label: UILabel!
+    @IBOutlet weak var usernameTextField: UITextField!
+    @IBOutlet weak var passwordTextField: UITextField!
+    @IBOutlet weak var button: UIButton!
+    @IBOutlet weak var saveCredsSwitch: UISwitch!
+    @IBOutlet weak var savePasswordLabel: UILabel!
+
+    private enum Constants {
+        static let userNameKey = "userNameKey"
+        static let passwordKey = "passwordKey"
+        static let isCheckedKey = "isCheckedKey"
+
+        //simulate API user
+        static let username = "user"
+        static let password = "password"
+    }
+
+    private var username: String? = nil
+    private var password: String? = nil
+    private var isChecked = false
+
+    @IBAction func saveCredsSwitchChanged() {
+        //update UD isCheckedKey
+        //if OFF
+        // remove userNameKey & passwordKey
+        if saveCredsSwitch.isOn == false {
```

Comment:
`saveCredsSwitch.isOn == false` the same as

`!saveCredsSwitch.isOn`

if no objections on rw swift style guide, use second option

Reply:
fixed